edx_rest_framework_extensions/auth/jwt/tests/test_cookies.py

```diff
@@ -46,6 +46,17 @@ def test_get_decoded_jwt_from_existing_cookie(self):
         self.assertEquals(expected_decoded_jwt, decoded_jwt)
 
     def test_get_decoded_jwt_when_no_cookie(self):
-        mock_request = mock.Mock(COOKIES={})
+        mock_request = mock.Mock(COOKIES={}, auth=None)
```

Comment:
I guess we won't need pass this since we are getting `getattr(request, 'auth', None` in `get_decoded_jwt` method.

Reply:
We need to mock it out otherwise a mock object is returned instead of a `None`. Since, `None or Mock()` would return `Mock()`.